.circleci/config.yml

```diff
@@ -463,7 +495,8 @@ workflows:
           <<: *after-linter
       - valgrind:
           test_args: CLUSTER=0 AOF=0
-          <<: *on-any-branch
+          #@@ <<: *on-any-branch
+          <<: *never
```

Comment:
Why disabling valgrind?

Reply:
Beat me to it. I'd argue that we should run valgrind on integration attempts not all branches -IMHO.

Reply:
`#@@` is disabling for the sake of the PR, it takes about an hour to run. Re-enabling it once everything passes.
Basically, running valgrind here is Dvir's call, I prefer to either invoke it manually or nightly or when a PR is merged.
We can also try to enable CLang Sanitizers on RedisAI.